kubehub/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from . models import CloudProvider
+
+
+class CloudProviderSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CloudProvider
+        fields = ('id', 'cp_type', 'name', 'api_endpoint', 'password')
+
+
+    def create(self, validated_data):
+        cloud_probider = CloudProvider.objects.create(**validated_data)
+        return cloud_probider
+
+
+    def update(self, instance, validated_data):
+        instance.cp_type = validated_data.get('cp_type', instance.cp_type)
```

Comment:
can we add validation as enum? like "PROX_MOX", "AWS"

Reply:
Yes, I just wanted to know full list of all providers types

Reply:
Fixed 8048aae